tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
+	"compilerOptions": {
+		"esModuleInterop": true,
+		"forceConsistentCasingInFileNames": true,
+		"lib": ["dom", "es6"],
+		"module": "commonjs",
+		"moduleResolution": "node",
+		"newLine": "lf",
+		"noFallthroughCasesInSwitch": true,
+		"noImplicitReturns": true,
+		"noUnusedLocals": true,
+		"noUnusedParameters": true,
+		"pretty": true,
+		"removeComments": true,
+		"sourceMap": true,
+		"strict": true
+	}
```

Comment:
I'm not sure about this but does it make sense to have this as template, instead of extending we can use the symlink?

Reply:
I think having the global setting and extending if they need to would be probably better considering in the future there will be lots of child packages and not all the project will have the same setup